pentaho-karaf-assembly/src/main/filtered-resources/etc-server/custom.properties

```diff
@@ -121,7 +121,9 @@ org.osgi.framework.system.packages.extra= \
  org.pentaho.di.engine.*;version="${pdi.version}", \
  org.reactivestreams.*;version="1.0.0", \
  org.pentaho.di.trans.dataservice.client.api.*;version="${pentaho-osgi-bundles.version}", \
- org.pentaho.di.trans.dataservice.jdbc.api.*;version="${pentaho-osgi-bundles.version}"
+ org.pentaho.di.trans.dataservice.jdbc.api.*;version="${pentaho-osgi-bundles.version}", \
```

Comment:
Should this ( https://github.com/pentaho/build-resources/blob/8.2.0.0/version.properties#L41 ) be the used version property here?

Reply:
the issue was not introduced in this PR. The rest of the PR is good 👍